From 08dda33acce9c1358b0cfb57e36bd621a5445fee Mon Sep 17 00:00:00 2001
From: Bartosz Golaszewski <bartosz.golaszewski@oss.qualcomm.com>
Date: Thu, 9 Apr 2026 10:27:13 +0200
Subject: [PATCH 14/29] NEWS: updates for v2.2.4

Add release notes for libgpiod v2.2.4.

Signed-off-by: Bartosz Golaszewski <bartosz.golaszewski@oss.qualcomm.com>
---
 NEWS | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/NEWS b/NEWS
index bd9edbb..f6b486b 100644
--- a/NEWS
+++ b/NEWS
@@ -2,6 +2,23 @@
 # SPDX-FileCopyrightText: 2017-2021 Bartosz Golaszewski <bartekgola@gmail.com>
 # SPDX-FileCopyrightText: 2023 Bartosz Golaszewski <bartosz.golaszewski@linaro.org>
 
+libgpiod v2.2.4
+===============
+
+Bug fixes:
+- fix buffer over-read bugs when translating uAPI structs to library types
+- fix variable and argument types where necessary
+- sanitize values returned by the kernel to avoid potential buffer overflows
+- fix memory leaks in gpio-tools
+- add missing return value checks in gpio-tools
+- fix period parsing in gpio-tools
+- use correct loop counter in error path in gpio-manager
+
+Improvements:
+- make tests work with newer coreutils by removing cases checking tools'
+  behavior on SIGINT which stopped working due to changes in behavior of the
+  timeout tool
+
 libgpiod v2.2.3
 ===============
 
-- 
2.53.0

