From 86583be89908202b137ec35350245b5f263a00c4 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Fri, 9 Jan 2026 18:02:48 -0600
Subject: [PATCH] pvr: disable PVR_I_WANT_A_BROKEN_VULKAN_DRIVER check

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 src/imagination/vulkan/pvr_physical_device.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/src/imagination/vulkan/pvr_physical_device.c b/src/imagination/vulkan/pvr_physical_device.c
index 46eb4e4..a2af8a7 100644
--- a/src/imagination/vulkan/pvr_physical_device.c
+++ b/src/imagination/vulkan/pvr_physical_device.c
@@ -1036,17 +1036,6 @@ VkResult pvr_physical_device_init(struct pvr_physical_device *pdevice,
       goto err_pvr_winsys_destroy;
 
    if (!pvr_device_is_conformant(&pdevice->dev_info)) {
-      if (!os_get_option("PVR_I_WANT_A_BROKEN_VULKAN_DRIVER")) {
-         result = vk_errorf(instance,
-                            VK_ERROR_INCOMPATIBLE_DRIVER,
-                            "WARNING: powervr is not a conformant Vulkan "
-                            "implementation for %s. Pass "
-                            "PVR_I_WANT_A_BROKEN_VULKAN_DRIVER=1 if you know "
-                            "what you're doing.",
-                            pdevice->dev_info.ident.public_name);
-         goto err_pvr_winsys_destroy;
-      }
-
       vk_warn_non_conformant_implementation("powervr");
    }
 
-- 
2.51.0

